src/GUI/ReverseEngineer20/packages.config

```diff
@@ -25,16 +25,20 @@
   <package id="Microsoft.Extensions.Logging.Abstractions" version="2.2.0" targetFramework="net461" />
   <package id="Microsoft.Extensions.Options" version="2.2.0" targetFramework="net461" />
   <package id="Microsoft.Extensions.Primitives" version="2.2.0" targetFramework="net461" />
-  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net461" />
+  <package id="MySqlConnector" version="0.49.2" targetFramework="net471" />
+  <package id="Newtonsoft.Json" version="11.0.2" targetFramework="net471" />
   <package id="Npgsql" version="4.0.4" targetFramework="net471" />
   <package id="Npgsql.EntityFrameworkCore.PostgreSQL" version="2.2.0" targetFramework="net471" />
+  <package id="Pomelo.EntityFrameworkCore.MySql" version="2.2.0" targetFramework="net471" />
+  <package id="Pomelo.JsonObject" version="2.2.0" targetFramework="net471" />
   <package id="Remotion.Linq" version="2.2.0" targetFramework="net461" />
   <package id="SQLitePCLRaw.bundle_green" version="1.1.11" targetFramework="net461" />
   <package id="SQLitePCLRaw.core" version="1.1.11" targetFramework="net461" />
   <package id="SQLitePCLRaw.lib.e_sqlite3.linux" version="1.1.11" targetFramework="net461" />
   <package id="SQLitePCLRaw.lib.e_sqlite3.osx" version="1.1.11" targetFramework="net461" />
   <package id="SQLitePCLRaw.lib.e_sqlite3.v110_xp" version="1.1.11" targetFramework="net461" />
   <package id="SQLitePCLRaw.provider.e_sqlite3.net45" version="1.1.11" targetFramework="net461" />
+  <package id="StrongNamer" version="0.0.8" targetFramework="net471" />
```

Comment:
Does this auto strongname a dll?

Reply:
Yes. The Pomelo.* are auto strongnamed by these. I was surprised it worked.

Reply:
Amazing!